boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -22,7 +22,8 @@
 #include <freeldr.h>
 #include <arch/pc/x86common.h>
 
-#define NDEBUG
+#undef DBG
```

Comment:
This is not how DBG and NDEBUG are used. DBG is global and specifies a DBG build. NDEBUG is local per file and controls verbosity of DPRINT (if NDEBUG is defined, only DPRINT1 is always printed, otherwise DPRINT is also printed). In freeldr we use a different approach by using TRACE/WARN/ERR and both components and levels. So NDEBUG should simply be removed.

Reply:
Thanks for the detailed explanation!
Done.